src/methods/chunk.rs

```diff
@@ -1,3 +1,94 @@
+//! Returns details of a specific chunk.
+//!
+//! You can use the [`block`](crate::methods::block) RPC method to get a valid chunk hash.
+//!
+//! ## Examples
+//!
+//! Chunks can be queried using one of two `ChunkReference` variants: `BlockShardId` and `ChunkHash`.
+//!
+//! 1. `BlockShardId`: Query a chunk by specifying its block and shard ID. Of which the `block_id` field is specified by either the block hash or height.
+//!    
+//!     a) Block hash:
```

Comment:
Going with the pattern established in https://github.com/near/near-jsonrpc-client-rs/pull/60/files, let's use an unordered list.

```suggestion
//!     - `BlockId::Hash`
```